WORKSPACE

```diff
@@ -6,96 +6,93 @@ load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 # EXTERNAL RULES
 http_archive(
     name = "com_github_grpc_grpc",
-    strip_prefix = "grpc-1.22.0",
+    sha256 = "d6af0859d3ae4693b1955e972aa2e590d6f4d44baaa82651467c6beea453e30e",
+    strip_prefix = "grpc-1.26.0-pre1",
     urls = [
-        "https://github.com/grpc/grpc/archive/v1.22.0.tar.gz",
+        "https://github.com/grpc/grpc/archive/v1.26.0-pre1.tar.gz",
```

Comment:
`pre` ? Does that mean it's an alpha or something ?  Do we need this specific release ?

Reply:
we do, this only builds with bazel 1.2.1